### ISSUE TITLE
[Enhancement] Import uninstalled games through account connection

### ISSUE THREAD

**Issue by scowalt** [Shawson/Playnite-OculusLibrary#14]
Similar to most of the other library sources, it would be nice to be able to connect to my Oculus account and import games that aren't installed locally.

I don't know if this is feasible. Maybe this is out-of-scope. But I think this would be a nice add.

Thanks for making this extension!

**Comment by Shawson** [comment]
I agree- this would be great!  At present, grabbing data from the Oculus store is a nightmare!  There' no officially supported api that I've seen yet so the little data I am grabbing (game title/ thumbnails) is scraped from the online store site! I'll take another look to see if any of this has changed since I initially wrote this..
